app/src/main/java/com/example/easysurvey/RetrofitClientInstance.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.easysurvey
+
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object RetrofitClientInstance {
+
+    private var retrofit: Retrofit? = null
+    private val BASE_URL = "https://example.survey.database/"
```

Comment:
Provided a sample url. This is where the url from your json file would be. Typically only the domain name portion of the url.

Reply:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.